Mathlib/Topology/ContinuousOn.lean

```diff
@@ -284,6 +284,11 @@ alias nhdsWithin_compl_singleton_sup_pure := nhdsNE_sup_pure
 @[simp]
 theorem pure_sup_nhdsNE (a : α) : pure a ⊔ 𝓝[≠] a = 𝓝 a := by rw [← sup_comm, nhdsNE_sup_pure]
 
+lemma continuousAt_iff_punctured_nhds
+    {α β : Type*} [TopologicalSpace α] [TopologicalSpace β] {f : α → β} {a : α} :
```

Comment:
Can you re-use the variables \alpha and \beta? In other words, making the part before the colon
`lemma continuousAt_iff_punctured_nhds [TopologicalSpace β] {f : α → β} {a : α} :`
should do.